feat(FE-98): Implemented Sliding Hero Page

terejoe opened the pull request:
I implemented a carousel using react and added an interval of three seconds to show the three different headers, texts and buttons. The three different buttons links to three different landing pages.
![Devask 1](https://user-images.githubusercontent.com/98318694/205487363-59f981f6-d60d-4345-a824-2732651b1d91.png)
![Devask 2](https://user-images.githubusercontent.com/98318694/205487367-934377fd-7e5a-4ddb-ad4f-2699c6013679.png)
![Devask 3](https://user-images.githubusercontent.com/98318694/205487368-63c1e289-f125-46b1-9258-20fc7bb5c8b7.png)

mrcoded reviewed (changes requested):
Please include detailed description of what you did and attach images to show

terejoe commented:
I implemented a carousel  using react and added an interval of three seconds to show the three different headers, texts and buttons. The three different buttons links to three different landing pages.

![Devask 1](https://user-images.githubusercontent.com/98318694/205486993-abe7ba1d-aacb-43d9-be19-2ed097a44fb6.png)
![Devask 2](https://user-images.githubusercontent.com/98318694/205487005-2722dbce-1328-4c90-88e4-442b1a6c7d6f.png)
![Devask 3](https://user-images.githubusercontent.com/98318694/205487112-84305255-b20c-4624-85c2-7e38eb0aebf1.png)
